src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+.html {
+  --header-link-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+}
+
+.img {
+  display: block;
+}
```

Comment:
You are using a tag name for styling which is not recommended. It's better to style elements using '.class', ':pseudo-class', 'pseudo-element' and '[attribute]'. (CHECKLIST: Don't use tag names for styling (except 'html' and 'body'))